src/posts/2022-11-30-gsod/index.md

```diff
@@ -0,0 +1,117 @@
+
+## **Problem Statement**
+
+Cloud native networking and security are complex topics that even long time practitioners struggle to master. Cilium documentation has grown organically over time and the project has mainly been used by advanced cloud native users. The documentation lacks the structure that new, and even existing, users can rely on to quickly find the information they need. In addition, users sometimes struggle to discover all of the features and functionality of Cilium because the documentation does not make them easily discoverable. Finally, new users often have questions about how to troubleshoot setting up their network.
+
+Improving the structure of the documentation will help new users adopt Cilium and reduce the support load on the community.
+
+## **Project Description**
+
+### **Creating the proposal**
+
+The Cilium team knew about Season of Docs from other CNCF projects that had participated in the past. We had some open PRs about restructuring the documentation, but nobody was finding the time to finish them. Two people from the Cilium team (Liz and Bill) volunteered to create a project [proposal](https://docs.cilium.io/en/v1.12/contributing/development/season_of_docs/).
+
+Once the proposal was written, we shared it in the weekly developer meeting for feedback. People were really excited about finally having someone to look at the overall docs structure! We made a PR of the proposal in the docs and had it approved by the maintainers. Finally, we opened our Open Collective account to participate and receive payments.
+
+#### **Budget**
+
+We estimated that hiring a part time writer would cost roughly $1,500 a month which helped us come up with our $9,000 project budget. We also included some extra to buy our writer Cilium swag to show what they did for the project.
+
+After our interviews, we really liked two of the applicants and were having a hard time choosing between them because they would each bring different things to the project. We ultimately asked them if they would be willing to split the budget to each work on different parts of the project. Both writers agreed so we moved forward with a split budget with each writer working on a different part of the project proposal. We liked what they each brought to the project and would recommend it for other projects faced with a similar decision.
+
+### **Participants**
+
+The core team working on this project was:
+
+* Liz Rice (Cilium)
+* Quentin Monnet (Cilium)
+* Bill Mulligan (Cilium)
+* Yoyo Wu (Technical Writer)
+* Divine Odazie (Developer Relations)
```

Comment:
Thank you very much, @xmulligan. I sincerely appreciate the opportunity. It changed my life. ❤️

God bless you 🙏🏾